# Goal definition: Distance to travel
float32 distance
---

# Result definition: Status code of the aciton
int8 status
uint64 rfid_code

---

# No feedback message, current implementation by ROS is redundant
